### PR TITLE
Policy replaces customer/home

### PR DESCRIPTION
**Pull request by hobbitronics**
## Removed
- customer and policy home routes
- customer home pages

## Changed
- "Policy Detail" to "Policy"

## Added
- CardGrid to "Policy" view
- More changes to follow https://trello.com/c/scelXA7p/247-home-%E2%86%92-policy

image

### CODE REVIEW

File: src/pages/home.svelte [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
-import { ADMIN_HOME, CUSTOMER_HOME } from 'helpers/routes'
+import user, { getDefaultPolicyId, User, UserAppRole } from '../authn/user'
+import { selectedPolicyId } from '../data/role-policy-selection'
+import { ADMIN_HOME, policyDetails } from 'helpers/routes'
 import { redirect } from '@roxi/routify'
 import { roleSelection } from 'data/role-policy-selection'
 
 $: sendToRoleHome($roleSelection)
+
+const redirectToAppropriatePolicy = (user: User, selectedPolicyId: string): void => {
```
Comment (RussellChamp) [inline src/pages/home.svelte:10]: just a minor nit-pick but it's nice to organize functions in the (approximate) logical order in which they will be invoked.

**Review by RussellChamp — APPROVED** [review]
lgtm

File: src/pages/policies/[policyId].svelte [hunk 2]
```diff
@@ -1,14 +1,16 @@
 <script lang="ts">
-import { ClaimsTable } from 'components'
-import { claimIsOpen } from 'data/claims'
+import { isAdmin } from '../../authn/user'
+import { CardsGrid, ClaimsTable, Row } from 'components'
+import { Claim, claimIsOpen, loadClaimsByPolicyId, selectedPolicyClaims } from 'data/claims'
```
Comment (Schparky) [inline src/pages/policies/[policyId].svelte:4]: another minor nit, but some of these lines aren't sorted by file path

**Review by Schparky — APPROVED** [review]

File: src/pages/home.svelte (hunk 1 above)
Comment (hobbitronics) [reply src/pages/home.svelte:10]: that's backwards from vanilla js needing to declare them for use before they're invoked (when using const) but I see the logic in that.

**Pull request merged**